analytics/lib/counts.py

```diff
@@ -811,6 +838,11 @@ def get_count_stats(realm: Optional[Realm] = None) -> Dict[str, CountStat]:
             CountStat.DAY,
             interval=TIMEDELTA_MAX,
         ),
+        CountStat(
+            "used_upload_space::day",
```

Comment:
We probably want something like `upload_quota_used_bytes`, if that's not too long? 

It's a `models.BigIntegerField`, so we don't have to worry about overflows.

Reply:
Yeah that works as a name. Something i tried to use earlier was too long so i made it super short, but this is better, so changed now.